src/geocoder/TECH_DEBT__TemporarilyCopied__Geocoder.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GovUk.Education.SearchAndCompare.Geocoder
+{
+
+    /// <summary>
+    ///    Tech debt: this is copy-and-pasted from Geocoder.cs in search-and-compare-ui.
+    ///    We should extract a utilities project these two can share.
+    ///    https://trello.com/c/66mYU1OF/308-tech-debt-extract-geocoder-into-shared-utilities-nuget-package
+    /// </summary> 
+    public class TECH_DEBT__TemporarilyCopied__Geocoder
+    {
+        private readonly string apiKey;
+
+        public TECH_DEBT__TemporarilyCopied__Geocoder(string apiKey)
+        {
+            this.apiKey = apiKey;
+        }
+
+        public async Task<Coordinates> ResolvePostCodeAsync(string postCode)
+        {
+            using(var client = new HttpClient())
+            {
+                var query = HttpUtility.ParseQueryString(string.Empty);
+                query["key"] = apiKey;
+                query["region"] = "uk";
+                query["address"] = postCode;
+
+                var response = await client.GetAsync("https://maps.googleapis.com/maps/api/geocode/json?" + query.ToString());
```

Comment:
magic string 

in config please

Reply:
i'm reluctant to make this diverge too much from the real Geocoder... :/